site/src/sphinx/server-annotated-service.rst

```diff
@@ -263,7 +263,9 @@ Getting an HTTP header
 
 Armeria also provides :api:`@Header` annotation to inject an HTTP header value into a parameter.
 The parameter annotated with :api:`@Header` can also be specified as one of the built-in types as follows.
-:api:`@Default` and ``Optional<?>`` are also supported.
+:api:`@Default` and ``Optional<?>`` are also supported. Unlike :api:`@Param` annotation, :api:`@Header`
+annotation supports ``List`` or ``Set`` as its type because HTTP headers can be added several times with the
```

Comment:
I'm curious if we need to support `List` or `Set` for query strings, e.g. `?foo=1&foo=2` could be translated into `@Param("foo") List<String>`.